src/meson.build

```diff
@@ -19,6 +19,10 @@ daemon_dep = [
   cbor,
 ]
 
+if get_option('passim').allowed()
+  daemon_dep += passim
```

Comment:
this seems like wrong place to put it; you want it in fwupdtool too.

Reply:
Ohh, the variable is badly named; it should be engine_dep -- it's used in both. I'll rename it in this PR.